### DISCUSSION TITLE
How to avoid `<IPython.core.display.Javascript object>` after cell execution in Jupiter Notebook

### DISCUSSION THREAD

**Discussion by 25shmeckles** [Textualize/rich#1881] [Q&A]
I am using `rich` extension for Jupiter Notebook as described in the Docs with `%load_ext rich`
All works fine but `<IPython.core.display.Javascript object>` gets printed after the execution of each cell. I understand this is the expected output since the object is actually returned as the cell output, but it is rather unpleasant to have it over and over through the notebook.
Is there a way to get rid of that output line?

<img width="1191" alt="Screenshot 2022-01-28 at 15 14 13" src="https://user-images.githubusercontent.com/25845049/151562524-126052cf-7772-4fb5-bf2f-22be2fb6e7be.png">

**Comment by willmcgugan** [comment]
Is this on vscode?

**Reply by 25shmeckles** [reply]
No, standard Jupiter Notebook

**Reply by willmcgugan** [reply]
Your code works fine here. Possibly some difference in software versions or your kernel.

**Reply by 25shmeckles** [reply]
It is a plain ipython kernel. I will check with others. But just to be
clear, that line isn't supposed to be there, correct?
